integration-tests.mdx

```diff
@@ -8,12 +8,14 @@ import { highlightedCode as integrationTest } from '@/samples/features/integrati
 
 ---
 
-Chalk resolvers are callable and [unit-testable](/docs/unit-tests) Python functions.
-However, you may want to test the interactions between Chalk
-resolvers with an integration test.
+In the previous section, we talked about how to [unit-testindividual resolvers](/docs/unit-tests)—this process is easy since each resolver is a callable Python functions and you can assert on the expected output. In this section we will talk about writing integration tests. Chalk provides two features that make integration testing really easy:
```

Comment:
```suggestion
In the previous section, we talked about how to [unit-test individual resolvers](/docs/unit-tests)—this process is easy since each resolver is a callable Python functions and you can assert on the expected output. In this section we will talk about writing integration tests. Chalk provides two features that make integration testing really easy:
```